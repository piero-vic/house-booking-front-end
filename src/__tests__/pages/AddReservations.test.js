import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../../redux/store';
import AddReservation from '../../pages/addReservation';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <Router>
          <AddReservation />
        </Router>
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
