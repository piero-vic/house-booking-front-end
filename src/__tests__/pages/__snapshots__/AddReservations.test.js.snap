// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<main
  className="min-h-screen min-w-full bg-center bg-cover bg-no-repeat bg-login"
>
  <div
    className="min-h-screen min-w-full grid place-items-center bg-lime-500 bg-opacity-80"
  >
    <div
      className="w-3/4 md:w-1/2"
    >
      <div
        className="border-b border-b-white p-5 mb-5 mx-auto w-10/12"
      >
        <h2
          className="text-white text-center font-bold text-4xl"
        >
          BOOK A RESERVATION
        </h2>
      </div>
      <form
        className="flex flex-col gap-5 items-center"
        onSubmit={[Function]}
      >
        <div
          className="w-full"
        >
          <input
            className="w-full bg-white py-2 px-4 rounded-3xl"
            name="date"
            onBlur={[Function]}
            onChange={[Function]}
            type="date"
          />
        </div>
        <input
          className="w-1/2 bg-lime-500 text-white py-2 px-4 border-2 border-white rounded-3xl font-bold"
          type="submit"
          value="Add Reservation"
        />
        <a
          className="w-1/2 bg-white text-lime-500 py-2 px-4 border-2 border-lime-500 rounded-3xl font-bold text-center"
          href="/reservations"
          onClick={[Function]}
        >
          Your Reservations
        </a>
      </form>
    </div>
  </div>
</main>
`;
